Markdown_to_TeX/markdown_compile.py

```diff
@@ -279,17 +270,20 @@ def convert(self):
             tex_contents = fle.read()
         
         #Replace template tags with relevant data.
-        tex_contents = tex_contents.replace(TITLE, title)
-        tex_contents = tex_contents.replace(TITLE, title)
-        tex_contents = tex_contents.replace(STUDENT_NAMES, authors)
-        tex_contents = tex_contents.replace(STUDENT_EMAILS, emails)
-        tex_contents = tex_contents.replace(ABSTRACT, abstract_latex)
+        if abstract_latex != "":
+            tex_contents = tex_contents.replace(ABSTRACT, abstract_latex)
         tex_contents = tex_contents.replace(SECTION_START, sections_latex)
 
+        for key in tag_key_values:
+            times_to_replace = tex_contents.count(key)
+            for i in range(0, times_to_replace):
+                tex_contents = tex_contents.replace(key, tag_key_values[key])
+
         if self.verbose:
             #Output complete LaTeX document.
             print("")
             print(tex_contents)
+            logging.debug(tex_contents)
         
  
         #Get rid of directory so we can regenerate the new .tex file.
```

Comment:
Line 298 (sorry, GitHub won't let me reference that line directly) is breaking on Windows Subsystem for Linux. WSL uses Unix style paths `like/this/` but Windows prefers `paths\\like\\this`. 
```
$ python Markdown_to_TeX/markdown_compile.py ./Markdown_to_TeX/Templates/markdown_template.md -t ./Markdown_to_TeX/Templates/PDD_Template.tex
Traceback (most recent call last):
  File "Markdown_to_TeX/markdown_compile.py", line 365, in <module>
    tex_path = latex.convert()
  File "Markdown_to_TeX/markdown_compile.py", line 299, in convert
    with open(tex_path, "w") as fle:
FileNotFoundError: [Errno 2] No such file or directory: './Markdown_to_TeX/Templates/markdown_template\\./Markdown_to_TeX/Templates/markdown_template.tex'
```

A solution to this is using `os.path.join()` instead of math operators on this string

Reply:
Want to do this in a seperate PR

Reply:
Want to cover that in a cross platform branch.